aspect/testing/rules/BUILD

```diff
@@ -27,7 +29,7 @@ java_binary(
         ":intellij_aspect_test_fixture_java_proto",
         "//aspect/testing:guava",
         "//proto:intellij_ide_info_java_proto",
-        "@com_google_protobuf_java//:protobuf_java",
+        "@com_google_protobuf//:protobuf_java",
```

Comment:
This is probably added through coypbara transforms, I'll need to change this from our coypbara config.